### DISCUSSION TITLE
Welcome to flowbite-vue Discussions!

### DISCUSSION THREAD

**Discussion by zoltanszogyenyi** [themesberg/flowbite-vue#73] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by xeno** [comment]
Hi everyone, I'm Jimmy. Been building websites since 1993. Went professional at age 17 in 1998.

My plan is to contribute to flowbite-vue regularly and help out with support and direction.

**Comment by cogor** [comment]
Hi folks, my name is Ilya. 
I work at frontend more than five years, not so long like Jimmy 😁.
I plan to contribute to flowbite-vue often and use it for my pet projects.

**Comment by zoltanszogyenyi** [comment]
Welcome @xeno and @cogor!

Really happy to see the eagerness to contribute to the Flowbite Vue library.

I myself am working on a v2.0 release for the main library, so it's exciting to imagine how the ecosystem will look in a few months from now :)

Please ping me on Discord so that I can give you both a contributor role.

**Comment by Sqrcz** [comment]
Hey 👋

It's been a bit quiet around here for a while ;)

I'm Tomek... I've also started in late 90's 👴 I'm Vue fan since 2017 ✌️ and I would like to help this project grow.

I've already played a bit with it for the last couple of days... and I can see it's great potential... but also the long road ahead... for which I'm happy to join.
